.github/workflows/build_and_test_workflow.yml

```diff
@@ -27,81 +27,28 @@ env:
   OSD_SNAPSHOT_SKIP_VERIFY_CHECKSUM: true
 
 jobs:
-  build-lint-test-linux:
-    runs-on: ubuntu-latest
-    container:
-      image: docker://opensearchstaging/ci-runner:ci-runner-rockylinux8-opensearch-dashboards-integtest-v2
-      options: --user 1001
```

Comment:
@kavilla do we still need this custom image? or can we just use the default `ubuntu-latest` image now?

Reply:
we shouldn't now that we use the script to find the version but i am worried about a few weeks from now when Node 14 is EoLed and chromedriver drops it.

Reply:
+1 I don't think we necessarily have to use a custom image besides the time when the chromedriver isn't releasing a new version for chrome when github updates their virtual environments or as @AMoo-Miki mentioned.

i'm fine with this. One other benefit of using a custom image but since the github user `1001` isn't added to the image is that if that if we add the user to have access the dependencies the image can already have all the setup dependencies step in it. Like the image has yarn and java.